docs/index.md

```diff
@@ -66,9 +66,11 @@ In order to run a local demo of the Autonolas Contribute service:
 
```

Comment:
Please update version on tutorial to 0.9.1

Reply:
Done